decidim-core/app/mailers/concerns/decidim/multitenant_asset_host.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A module to be included in mailers that changes the default behaviour so
+  # the emails are rendered in the user's locale instead of the default one.
+  module MultitenantAssetHost
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :set_asset_host
+
+      # set_asset_host allows to redefine the host asset if not specified in application configuration
+      def set_asset_host
+        return if Rails.application.config.action_mailer.asset_host.present?
+
+        self.asset_host = ->(_mail) { "#{use_https? ? "https" : "http"}://#{@organization.host}" }
+      end
+
+      private
+
+      def use_https?
+        Rails.env.production? || Rails.configuration.force_ssl
```

Comment:
I know It won't be usual, but if the production site has no force_ssl enable, will this enforce https in assets anyway?

Reply:
Also, #5553 introduced a custom `force_ssl` in Decidim core, maybe we should use that.

Reply:
Done!